content/source/docs/enterprise/api/modules.html.md

```diff
@@ -196,16 +238,39 @@ curl \
 
 ## Create a Module Version
 
-Creates a new registry module version. After creating the version, the module should be uploaded to the returned upload link.
+`POST /registry-modules/:organization_name/:name/:provider/versions`
 
-| Method | Path |
-| :-- | :-- |
-| `POST` | `/registry-modules/:organization_name/:name/:provider/:versions` |
+Parameter            | Description
+---------------------|------------
+`:organization_name` | The name of the organization to create a module in. The organization must already exist, and the token authenticating the API request must belong to the "owners" team or a member of the "owners" team.
+`:name`              | The name of the module for which the version is being created.
+`:provider`          | The name of the provider for which the version is being created.
 
-### Parameters
+Creates a new registry module version. This endpoint only applies to modules without a VCS repository; VCS-linked modules automatically create new versions for new tags. After creating the version, the module should be uploaded to the returned upload link.
```

Comment:
@justincampbell What will the behavior be if someone attempts this on a VCS-backed module?

Reply:
It will still create a new version, and the version will be awaiting upload to the registry.

Reply:
@justincampbell based on the conversation last week where this could cause conflicts, shall I leave my edit as-is?